jsapp/js/components/permissions/sharingForm.es6

```diff
@@ -116,7 +116,7 @@ class SharingForm extends React.Component {
           {this.state.asset.name}
         </bem.Modal__subheader>
 
-        {stores.session.currentAccount.extra_details?.require_auth !== true &&
+        {stores.session.currentAccount.extra_details?.require_auth !== true && asset_type == 'survey' &&
```

Comment:
Instead of using literal `'survey'`, please use `ASSET_TYPES.survey.id` (coming from `js/constants`, already being used in this file) :) When fixed, feel free to merge